src/command_manager.cpp

```diff
@@ -82,9 +82,22 @@ void CommandManager::param_change_callback(uint16_t param_id)
 
 void CommandManager::init_failsafe()
 {
-  multirotor_failsafe_command_.F.value = RF_.params_.get_param_float(PARAM_FAILSAFE_THROTTLE);
+  float failsafe_thr_param = RF_.params_.get_param_float(PARAM_FAILSAFE_THROTTLE);
+  bool fixedwing = RF_.params_.get_param_int(PARAM_FIXED_WING);
+  if (!fixedwing
+      && (failsafe_thr_param < 0. || failsafe_thr_param > 1.0)) // fixed wings always have 0 failsafe throttle
```

Comment:
I don't think we need to enforce this.  Our mantra so far has been "we are all adults here", so I'm fine if someone wants a non-zero failsafe throttle if they're using a fixedwing

Reply:
No, this allows for the failsafe not to be configured for fixedwings, because it is ignored.